source/driver/Driver.cpp

```diff
@@ -143,6 +154,10 @@ void Driver::addStandardArgs() {
 
     cmdLine.setPositional(
         [this](string_view fileName) {
+            if (fileName.rfind(".e"sv) == fileName.size() - 2)
```

Comment:
I get what you're trying to do here but it's a little too ad-hoc and project-specific for my liking. Perhaps we can back this part of the change out for now and in a separate diff we can add a configureable filter to exclude files, maybe based on a glob or regex pattern or something.

Reply:
I would like to do this now, because the design I'm test-parsing includes vendor VHDL and DV files in Specman, and without filtering those, I won't be able to complete it.
What about something like `--exclude-ext e,vhd` (or maybe `--noext` , to be more in-line with `--libext`)  and use the `std::vector<std::string>&` option type that already exists to store all the extensions to be skipped.

Reply:
Yeah, I like --exclude-ext